opentelemetry/src/main/scala/zio/telemetry/opentelemetry/tracing/Tracing.scala

```diff
@@ -414,6 +414,12 @@ trait Tracing { self =>
     trace: Trace
   ): UIO[Span]
 
+  def setErrorStatus[E <: Throwable](
```

Comment:
The subtyping annotation `<: Throwable`  is redundant here.

Reply:
fixed